test/e2e/plugins/server-info.go

```diff
@@ -32,13 +32,13 @@ var _ = ginkgo.Describe("server-info plugin", func() {
 	}
 	s := scaffold.NewScaffold(opts)
 
-	ginkgo.It("enable server-info plugin", func() {
+	ginkgo.It("disable server-info plugin", func() {
```

Comment:
There is no need to adjust the content of the test case, just Skip() the test case directly